strats.md

```diff
@@ -516,6 +520,23 @@ A `comeInWithDoorStuckSetup` condition must match with a `leaveWithRunway` condi
 }
 ```
 
+### Come In Speedballing
+
+A `comeInSpeedballing` entrance condition indicates that Samus must enter the room either in a speedball from the previous room, or in a process of running, jumping, or falling into a speedball. It has the following property:
+
+- _runway_: A [runway geometry](#runway-geometry) object describing the tiles available in the current room to complete the speedball. The end of this runway represents the point by which the speedball must be complete (i.e. when Samus must be morphed and on the ground with blue speed). A runway length of 0 would represent that the speedball must be completed before the transition.
+
+It is assumed that the runway in the current room is level or sloping up; adjustments would be needed to handle a case where it sloped down.
+
+Note that a `comeInSpeedballing` entrance condition can always be satisfied by coming into the room while already in a speedball. Therefore, a different entrance condition must be used if the strat specifically requires obtaining the speedball after entering the room (either by jumping through the transition, or by running through the transition and jumping afterward).
+
+A `comeInSpeedballing` entrance condition must match with a `leaveWithRunway` condition on the other side of the door. A match with a `leaveWithRunway` comes with implicit requirements:
+- A `canShineCharge` based on the combined runway length, minus the amount of tiles needed to perform the jump into the speedball. The amount of tiles needed for the jump depends on the player's shortcharge ability (as well as ability to short-hop mockball): obtaining blue speed with lower run speed means less space needed to perform the jump. This can be approximated in a simple way with the following assumptions:
+  - If the tech `canSlowShortCharge` is enabled, then 5 tiles are needed for the jump into the speedball.
+  - Otherwise 14 tiles are needed.
```

Comment:
Seems slick, and can easily be expanded to include more precision if we ever want it. But I'm no expert on this.